pkg/trace/config/config.go

```diff
@@ -127,7 +127,7 @@ func New() *AgentConfig {
 		Endpoints:  []*Endpoint{{Host: "https://trace.agent.datadoghq.com"}},
 
 		BucketInterval:   time.Duration(10) * time.Second,
-		ExtraAggregators: []string{"http.status_code", "version"},
+		ExtraAggregators: []string{"http.status_code", "version", "_dd.hostname"},
```

Comment:
@jdgumz and @pkalmakis having the underscore prefix on the tags for these stats is fine? it doesn't get normalized/stripped or anything in the pipeline?

Reply:
We've already tried this. It gets sent as is to the backend.